### ISSUE TITLE
Discrepancy with BLIP paper results when using PyTorch > 1.10

### ISSUE THREAD

**Issue by oscmansan** [salesforce/LAVIS#40]
I was trying to reproduce results with BLIP on VQAv2 test-dev and I observed a non-negligible difference between the VQA accuracy obtained using the [published checkpoint](https://storage.googleapis.com/sfr-vision-language-research/BLIP/models/model_base_vqa_capfilt_large.pth) (**77.41%**) and the number reported in the [paper](https://arxiv.org/abs/2201.12086) (**78.25%**).

These are the steps I followed:
1. Clone this repo
2. Install dependencies with `pip install .`
3. Create a symlink `cache/coco/images` pointing to the local copy of the COCO images
4. Modify `lavis/projects/blip/eval/vqav2_eval.yaml` as follows:
![image](https://user-images.githubusercontent.com/11041768/200872220-ecca0e39-e98b-465c-98b4-2e8a9a78cde5.png)
5. Run `python -m torch.distributed.run --nproc_per_node=4 evaluate.py --cfg-path lavis/projects/blip/eval/vqav2_eval.yaml` (note I only have 4 A100 GPUs available)
6. Submit the `test_vqa_result.json` file generated in `lavis/output/BLIP/VQA/...` to [EvalAI](https://eval.ai/web/challenges/challenge-page/830/submission)

After some debugging, I narrowed it down to a discrepancy in PyTorch versions: I was using the latest version (**1.13.0**), while LAVIS fixes the version to **1.10.0**. So there is some change between PyTorch 1.10 and PyTorch 1.13 which causes a performance degradation when loading a checkpoint trained on 1.10. After downgrading the PyTorch version to 1.10.0, I am able to achieve **78.24%** VQA accuracy on VQAv2 test-dev, almost the same number reported in the paper.

**Comment by HerocatUED** [comment]
I wonder how to install successfully with torch >1.10.0
There always is an error about torch vision
![2023-01-18](https://user-images.githubusercontent.com/104823035/213072152-c5341eae-1552-47aa-b9d2-b8359e5e4ca9.png)

**Comment by stlmx** [comment]
I got the same result (77.41%) with you and I will try torch 1.10 now.

**Comment by stlmx** [comment]
When using pytorch1.10, right result can be got.
